pkg/vere/io/http.c

```diff
@@ -2974,7 +2992,8 @@ _http_io_exit(u3_auto* car_u)
   u3h_free(htd_u->sax_p);
   u3h_free(htd_u->nax_p);
 
-  munmap(htd_u->stk_u, u3a_page);
+  if ( NULL != htd_u->stk_u )
+    munmap(htd_u->stk_u, u3a_page);
```

Comment:
u3 style is to always uses braces (except for control flow on the same line as the if).

Reply:
done